### DISCUSSION TITLE
Optimal number of CPU cores

### DISCUSSION THREAD

**Discussion by brandon-holt** [emdgroup/baybe#444] [Q&A]
Do you know if there is an optimal number of CPUs to run baybe on, or should I just use the max number of cores available?

**Comment by AdrianSosic** [comment]
Hi @brandon-holt. I see no reason not to use whatever is available. But perhaps I'm missing an important part of the question? 🙃

**Comment by Scienfitz** [comment]
FYI
I converted this into a discussion as it appears not to describe a problem or feature request

**Comment by AVHopp** [comment]
Is this still relevant? It's more than a year since the last comment here, so can this be closed?
